DESCRIPTION

```diff
@@ -34,7 +34,7 @@ Imports:
     knitr,
     magrittr,
     mvtnorm,
-    naomi.options (>= 0.1.2),
+    naomi.options (>= 0.1.5),
```

Comment:
Should this be 0.1.6? Consistent with https://github.com/mrc-ide/naomi.options/pull/21/files#diff-9cc358405149db607ff830a16f0b4b21f7366e3c99ec00d52800acebe21b231c